integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -561,6 +562,9 @@ def move_page(request, region_slug, language_slug, page_id, target_id, position)
         messages.error(request, e)
         logger.exception(e)
 
+    for page in Page.objects.filter(region=region):
+        invalidate_obj(page)
+
```

Comment:
```suggestion
```
Again, I don't think this changes anything, because the page model inherits from abstract tree node and will get invalidated during move anyway:
https://github.com/digitalfabrik/integreat-cms/blob/8c06962b2549f3c6839bc38b4b8f25d95a6ce7e6/integreat_cms/cms/models/abstract_tree_node.py#L296-L300
We could think about invalidating only the region's tree nodes there, but actually I think this isn't enough because moving a root page inside a region can change all tree_ids of all other regions as well.

Reply:
Ah sorry, I misunderstood the implications.